.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
```

Comment:
I added it in just to have one version of `go` below the the last stable release. I can add this to the builds that are [allowed to fail](http://docs.travis-ci.com/user/build-configuration/#Rows-That-are-Allowed-To-Fail) so that our build doesn't fail for backwards compatibility if you prefer but I think it's nice to have at least one version behind the latest.